README.md

```diff
@@ -131,12 +131,16 @@ Some of these may be seem redundant but are specified so future users can overri
 | checkmk_agent_group | Name of the group that should own the CheckMK Agent executable file | `"{{ checkmk_agent_user }}"` |
 | checkmk_agent_home | Full pathname of the CheckMK Agent user home folder | `"/home/{{ checkmk_agent_user }}"` |
 | checkmk_agent_local_checks | List of checks to copy to the "local" folder | `"count_users"` `"count_zombie_procs"` |
+| checkmk_agent_local_checks_async | List of checks to copy to the "local" async folders | https://docs.checkmk.com/latest/en/localchecks.html#cache |
```

Comment:
The format of the table is as shown in the header line.  The third column is for the default setting, not references.  Please review the surrounding lines and update to match?

Reply:
I fixed that in my last commit.